app/src/main/java/com/morihacky/android/rxjava/fragments/BaseFragment.java

```diff
@@ -13,4 +14,8 @@ public void onDestroy() {
         RefWatcher refWatcher = MyApp.getRefWatcher();
         refWatcher.watch(this);
     }
+
+    protected boolean _isCurrentlyOnMainThread() {
```

Comment:
should probably remove the _ prefix, since it's no longer private to that class/example.